api/UIEvent.json

```diff
@@ -5,34 +5,34 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/UIEvent",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": true
           },
           "chrome": {
-            "version_added": null
+            "version_added": true
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": true
           },
           "edge": {
-            "version_added": null
+            "version_added": true
           },
           "edge_mobile": {
-            "version_added": null
+            "version_added": true
           },
           "firefox": {
-            "version_added": null
+            "version_added": true
           },
           "firefox_android": {
-            "version_added": null
+            "version_added": true
```

Comment:
Add : 

`"notes": "The <code>isChar</code> property has never been supported by any browser but Firefox, and even on Firefox it's never worked except on Mac OSX. For that reason, it's been removed in Firefox 55 to align with other browsers."` ?